graph/Cargo.toml

```diff
@@ -11,7 +11,8 @@ atomic_refcell = "0.1.13"
 # Make sure no newer version of bigdecmal library is used in diesel dependencies reflected in the Cargo.lock file.
 # Automatic reneratiion of the Cargo.lock file would break compilation as only diesel dependencies would be updated.
 # Using a higher version of bigdecimal in complete graph-node project would change the PoI, hence break the consensus.
```

Comment:
I guess we can remove the comment in the three lines above, since this PR also solves exactly that technical debt.

Reply:
I rewrote that comment

Reply:
That's even better!